crates/biome_js_syntax/src/expr_ext.rs

```diff
@@ -284,6 +290,22 @@ impl JsBinaryExpression {
             Ok(false)
         }
     }
+
+    pub fn get_node_position(&self, node: &JsSyntaxNode) -> Option<BinaryExpressionNodePosition> {
```

Comment:
Coudl you add a documentation and doctest?

Reply:
Sure, thanks